everyvoice/demo/app.py

```diff
@@ -358,10 +276,15 @@ def create_demo_app(
                 btn = gr.Button("Synthesize")
             with gr.Column():
                 out_audio = gr.Audio(format="wav")
-                out_file = gr.File(label="File Output")
+                if output_list == ["wav"]:
```

Comment:
Maybe best to use `SynthesizeOutputFormats.wav` here instead of the `wav` string. I doubt this will ever change, but just in case.

Reply:
I'll test if that works, if so sure, it's the same either way to me.
I'm not generally a fan of how long references to python enum values are, but it still makes sense to use them systematically when we define them.